source/reference/mongosync.txt

```diff
@@ -176,6 +176,38 @@ Global Options
 
    Prints ``mongosync`` version information to stdout.
 
+.. _c2c-cli-oidc:
+
+Microsoft Azure Managed Identities Environment Variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/mongosync-and-oidc.rst
+
+To use Microsoft Azure Managed Identities for federated authentication,
+define these environment variables:
+
+.. option:: AZURE_TENANT_ID <id>
+
+   Sets the Azure tenant identifier. Example:
+   ``AZURE_TENANT_ID=08206ab8-16a0-406d-85e4-2f15f5620fac``
+
+.. option:: AZURE_APP_CLIENT_ID <id>
+
+   Sets the Azure application client identifier. Example:
+   ``AZURE_APP_CLIENT_ID=b6c835da-e536-425b-9405-64bc471e245b``
+
+.. option:: AZURE_CLIENT_ID <id>
+
+   Sets the Azure client identifier. Example:
+   ``AZURE_CLIENT_ID=f176d4eb-7dcd-4f66-bccf-aaa316ee61fd``
+
+.. option:: AZURE_FEDERATED_TOKEN_FILE <file>
+
+   Sets the Azure federated token file and path. Example:
+   ``AZURE_FEDERATED_TOKEN_FILE=/var/run/secrets/azure/tokens/azure-identity-token``
+
+For a complete example, see
+:ref:`c2c-conn-oidc2oidc-azure-managed-identities-example`.
 
```

Comment:
Are there no environment variables needed for GCP? If not, it might be good to add a quick note to that effect.

Reply:
Correct